pkg/gofr/http/middleware/oauth.go

```diff
@@ -26,6 +26,8 @@ type authMethod int
 
 const (
 	JWTClaim authMethod = iota // JWTClaim represents the key used to store JWT claims within the request context.
+	Username authMethod = 1    // Username fetches the username when we use Basic Auth
```

Comment:
These constants are not part of the OAuth, please declare them where they are used.

Reply:
Also it's iota starting at 1, so the constant would have same value as previous variable?